src/UnauthenticatedApp.tsx

```diff
@@ -7,17 +7,49 @@ import TopNav from './components/widgets/AppTopNav'
 import {setBodyClass} from './helpers/utility'
 import PublicRoutes from './routes_public'
 import constants from './types/constants'
+import ArcLogo from './assets/arc/arc_main_logo.svg'
```

Comment:
do we need special directory for arc? I think it's just a logo. Let's keep it in the root of /assets